frontend-react/src/components/Table/Table.test.tsx

```diff
@@ -91,19 +86,21 @@ const selectDatesFromRange = async (dayOne: string, dayTwo: string) => {
     const endDatePickerButton = datePickerButtons[1];
 
     /* Select Start Date */
-    await userEvent.click(startDatePickerButton);
-    const newStartDateButton = screen.getByText(`${dayOne}`);
-    await userEvent.click(newStartDateButton);
+    userEvent.click(startDatePickerButton);
```

Comment:
Why were the `await`s removed for the `userEvent` interactions?  Aren't they all asynchronous now?

Reply:
I have to await on the following findBy so that side effects are settled